cmd/cluster-agent/webhook/listener.go

```diff
@@ -0,0 +1,23 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+/*
+Package api implements the agent IPC api. Using HTTP
+calls, it's possible to communicate with the agent,
+sending commands and receiving infos.
+*/
+package webhook
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+// getListener returns a listening connection
+func getListener() (net.Listener, error) {
```

Comment:
Can you get rid of this function and just move the command to where it is called? It doesn't make sense to have it because it is only called in one place.

Reply:
Addressed 👍